src/DevBetterWeb.Infrastructure/Services/GraduationCommunicationsService.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using DevBetterWeb.Core;
+using DevBetterWeb.Core.Entities;
+using DevBetterWeb.Core.Interfaces;
+using DevBetterWeb.Infrastructure.Identity.Data;
+using Microsoft.AspNetCore.Identity;
+
+namespace DevBetterWeb.Infrastructure.Services
+{
+  public class GraduationCommunicationsService : IGraduationCommunicationsService
+  {
+    private readonly IEmailService _emailService;
+    private readonly UserManager<ApplicationUser> _userManager;
+
+    public GraduationCommunicationsService(IEmailService emailService,
+      UserManager<ApplicationUser> userManager)
+    {
+      _emailService = emailService;
+      _userManager = userManager;
+    }
+
+    public async Task SendGraduationCommunications(Member member)
+    {
+      var memberEmail = "";
+      var memberSubject = "Congratulations from DevBetter!";
+      var memberText = "You're now an alumnus of DevBetter! Congratulations on your graduation!\nNow that you're an alumnus, you'll retain full access as long as you want it, but your subscription is free.";
```

Comment:
"as long as you want it (and you follow our code of conduct and other terms), ..."